mesonbuild/compilers/compilers.py

```diff
@@ -1804,3 +1831,88 @@ def get_optimization_args(self, optimization_level):
 
     def get_debug_args(self, is_debug):
         return clike_debug_args[is_debug]
+
+class CcrxCompiler:
+    def __init__(self, compiler_type):
+        if not self.is_cross:
+            raise EnvironmentException('ccrx supports only cross-compilation.')
+        # Check whether 'rlink.exe' is available in path
+        self.linker_exe = 'rlink.exe'
```

Comment:
did you want to resolve this todo?

Reply:
This is not a todo, the check is done just below this line